docs/guides/embeds.md

```diff
@@ -0,0 +1,129 @@
+# Embeds
+
+Embeds in Discord are just **embedded content messages** that you can use to **organize the response of the bot**, with **larger titles**, **images**, **footers**, and embed **colors**. As an added benefit, they have a higher message content limit on its description. Embeds can be used to construct clickable dialogs, for example.
```

Comment:
```suggestion
Discord's embeds stand for **embedded content messages** that can be used to present data with special formatting and structure. They have a higher message content character length limit as an added benefit and can be used to create clickable dialogs, for example.
```

Reply:
Just a question, can I add ** in `used to present data with special formatting and structure` and `higher message content character length limit`?